src/CarbonAware.CLI/test/CarbonAwareCLITests.cs

```diff
@@ -36,32 +36,33 @@ public void ParseCommandLineArguments_SetsStartTimeAndEndTimeWhenProvided()
     {
         string[] args = new string[] {"-l", "test", "-t", "2021-11-11", "--toTime", "2021-12-12"};
         
-        var cli = new CarbonAwareCLI(args, It.IsAny<ICarbonAwareAggregator>());
+        var cli = new CarbonAwareCLI(args, It.IsAny<ICarbonAwareAggregator>(), Mock.Of<ILogger<CarbonAwareCLI>>());
 
         Assert.AreEqual("test", cli._state.Locations[0]); 
         Assert.AreEqual(DateTime.Parse("2021-11-11"), cli._state.Time); 
         Assert.AreEqual(DateTime.Parse("2021-12-12"), cli._state.ToTime); 
     }
 
-    [Test]
-    public void ParseCommandLineArguments_ThrowsErrorWhenLocationNotProvided()
-    {
-        string[] args = new string[] {};
-        var stringWriter = new StringWriter();
-        Console.SetOut(stringWriter);
-        new CarbonAwareCLI(args, It.IsAny<ICarbonAwareAggregator>());
-        StringAssert.Contains("Required option 'l, location' is missing.", stringWriter.ToString());
-    }
+    // Commenting out for now due to swallowed error
+    // [Test]
+    // public void ParseCommandLineArguments_ThrowsErrorWhenLocationNotProvided()
```

Comment:
Prefer: `[Ignore("Ignore for now due to swallowed error")]`  and leave this uncommented

Reply:
Made the change to ignore instead. I debugged for a bit today but couldn't figure out the issue, I reached out to @pritipath  who originally wrote it and she'll be taking a look to see if she knows what's going on. My guess from what I saw was that the exception that is usually thrown keeps getting swallowed, I'm guessing in the general exception handling code we added.

Reply:
Also this failure isn't specific to this change. I noticed that the CLI Tests weren't part of the solution and thus weren't being run as part of `dotnet test` so they were actually out of date regardless, so not quite sure which PR may have caused them to break originally.